exec/cnex/array.c

```diff
@@ -47,6 +70,30 @@ void array_freeArray(Array *self)
     free(self);
 }
 
+void array_clearArray(Array *self)
+{
+    if (self != NULL && self->data != NULL) {
+        for (size_t i = 0; i < self->size; i++) {
+            cell_clearCell(&self->data[i]);
+        }
+        free(self->data);
+    }
+    self->data = NULL;
+    self->size = 0;
+
+    array_expandArray(self, 0);
```

Comment:
What is this call intended to do? You've already cleared the array up to line 82, there really isn't anything else to do.

Reply:
This was to ensure that self->data was allocated, and not NULL.  But I suppose with all of the other code I have in the array handlers, it will always MAKE space, if there isn't any, so yea, I could see this call as being redundant.